src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -1,7 +1,8 @@
 package core.basesyntax.service;
 
+import core.basesyntax.exeption.InvalidUserException;
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws InvalidUserException;
```

Comment:
You made it checked exception, but README file says that you should create **unchecked** exception. So I propose you to remove **throws InvalidUserException** and refactor your custom exception class to extend **RuntimeException** instead
```suggestion
    User register(User user);
```